config/manifests/bases/freeipa-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,157 @@
+      displayName: IDM
+      kind: IDM
+      name: idms.idmocp.redhat.com
+      version: v1alpha1
+  description: Deploy Freeipa in OpenShift
+  displayName: Identity Management as a Service
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - config.openshift.io
+          resources:
+          - dnses
+          verbs:
+          - list
+          - get
+          - watch
+        serviceAccountName: idm-operator-controller-manager
+      deployments: []
+      permissions:
+      - rules:
+        - apiGroups:
+          - apps
+          resources:
+          - statefulsets
+          verbs:
+          - create
+          - get
+          - list
+          - update
+          - watch
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - create
+          - get
+          - list
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - create
+          - get
+          - update
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          verbs:
+          - create
+          - list
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - idmocp.redhat.com
+          resources:
+          - idms
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - idmocp.redhat.com
+          resources:
+          - idms/finalizers
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - idmocp.redhat.com
+          resources:
+          - idms/status
+          verbs:
+          - get
+          - patch
+          - update
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - create
+          - list
+          - watch
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes/custom-host
+          verbs:
+          - create
+        serviceAccountName: idm-operator-controller-manager
+    strategy: ""
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - freeipa
+  - kerberos
+  - pki
+  - sso
+  - idm
+  links:
+  - name: Freeipa Operator
+    url: https://github.com/freeipa/freeipa-operator
```

Comment:
Double check if this is the URL to display here. It is displayed into the operator information in OLM from the OpenShift console.